worker/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM openjdk:13
 WORKDIR /usr/src/ab2d-worker
 ADD target /usr/src/ab2d-worker
-CMD java -jar worker-*-SNAPSHOT.jar
\ No newline at end of file
+CMD java -javaagent:/usr/src/ab2d-worker/newrelic/newrelic.jar -jar worker-*-SNAPSHOT.jar
```

Comment:
In dev I assume we'll need to download newrelic and put the jar in that location?

Reply:
If you do the docker containers, it creates this directory within the docker container.

Reply:
Ahh ok..sometimes we run the applications from intellij, I'm thinking that it will just ignore the new relic annotations/API code that I put in for future dev if I don't run it with the javaagent and shouldn't cause an issue.

Reply:
But back to your original question, the following directories are created when you build:
- api/target/newrelic/
- worker/target/newrelic/
The contents of the newrelic directories that are created are:
- CHANGELOG
- LICENSE
- README.txt
- THIRD_PARTY_NOTICES.md
- extension-example.xml
- extension.xsd
- newrelic-api-javadoc.jar
- newrelic-api-sources.jar
- newrelic-api.jar
- newrelic.jar
- newrelic.yml
- nrcerts